### PR TITLE
docs: Replace old readme instructions

### PR DESCRIPTION
**Pull request by svanherk**
There's a lot of redundancy in this readme that could be cleaned up, but the goal of this PR is to try to remove the lies! 😈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,41 +112,21 @@ The `d2l-navigation-band` also includes a `slot` with a custom scrollbar and fad
 
 ### d2l-navigation-main-header
 
-Include the [webcomponents.js](http://webcomponents.org/polyfills/) "lite" polyfill (for browsers who don't natively support web components), then import `d2l-navigation-main-header.html`:
+Include the [webcomponents.js polyfill loader](http://webcomponents.org/polyfills/) (for browsers who don't natively support web components), then import `d2l-navigation-main-header.js`:
```
Comment (dlockhart) [inline README.md:115]: If you want you can just remove all talk of the polyfill -- I don't think we need it in individual repos anymore.

File: README.md [hunk 2]
```diff
@@ -225,41 +143,21 @@ Then add the `d2l-navigation-main-header`, and provide elements for the `left` a
...
 ### d2l-navigation-main-footer
 
-Include the [webcomponents.js](http://webcomponents.org/polyfills/) "lite" polyfill (for browsers who don't natively support web components), then import `d2l-navigation-main-footer.html`:
+Include the [webcomponents.js polyfill loader](http://webcomponents.org/polyfills/) (for browsers who don't natively support web components), then import `d2l-navigation-main-footer.js`:
 
 ```html
 <head>
-	<script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-	<link rel="import" href="bower_components/d2l-navigation/d2l-navigation-main-footer.html">
+  <script src="@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
```
Comment (dlockhart) [inline README.md:150]: Similarly, no point really in complicating the examples by including the polyfill.

File: README.md [hunk 3]
```diff
@@ -10,45 +10,25 @@ A series of [Polymer](https://www.polymer-project.org/1.0/)-based web components
 
 ### d2l-navigation
```
Comment (dlockhart) [inline README.md:12]: One more lie above ^, link goes to Polymer 1 documentation.

**Review by dlockhart — APPROVED** [review]

**Review by jeroach — APPROVED** [review]
Thanks for cleaning this up!

**Pull request merged**